README.md

```diff
@@ -17,7 +17,7 @@ user configuration opportunities. It is registered open source hardware with
 ### Open Source Chassis
 
 The Launch chassis is licensed CC-BY-SA-4.0 and can be viewed in the
-[chassis](./chassis/) folder using [FreeCAD](https://www.freecadweb.org/).
+[chassis](./chassis/) folder using [FreeCAD].
```

Comment:
Thank you for updating this link. While the collapsed reference link is a valid part of both GitHub Flavored Markdown and the CommonMark specification, I'm not sure it's adding to the document here.

Since we mention FreeCAD in this section, it doesn't seem great to have its reference listed 80 lines later. It seems like it would make more sense to have the reference listed at the end of this particular section (which should be valid, from what I see in the spec).

However, at that point, the reference is immediately after where this link is at, so it might as well still be a normal link.

Reply:
Sure